src/pages/NavbarPage.tsx

```diff
@@ -0,0 +1,131 @@
+import { FC } from 'react';
+import { CodeExample, DemoPage } from './DemoPage';
+import { Navbar } from '../components';
+import { Button } from '../components';
+
+const NavbarPage: FC = () => {
+  const examples: CodeExample[] = [
+    {
+      title: 'Default Navbar',
+      code: (
+        <Navbar>
+          <Navbar.Brand href="https://flowbite.com/">
+            <img src="https://flowbite.com/docs/images/logo.svg" className="mr-3 h-6 sm:h-9" alt="Flowbite Logo" />
+            <span className="self-center whitespace-nowrap text-xl font-semibold dark:text-white">Flowbite</span>
```

Comment:
This can probably be replaced with `Navbar.Link` in the future.

Reply:
I think `Navbar.Brand` is already fulfilling the option of having it as a link.

Reply:
Okay cool. Should I leave it as is?

Reply:
Yes I think so